src/Tmds.Ssh/Ed25519PrivateKey.cs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Buffers;
+using Org.BouncyCastle.Math.EC.Rfc8032;
+
+namespace Tmds.Ssh;
+
+sealed class Ed25519PrivateKey : PrivateKey
+{
+    private const int _privateKeySize = 32;
```

Comment:
nit: for a `const`, the convention is `PrivateKeySize`.

Reply:
Thanks I'll remember that for next time. Now that the array is already split in the parser it's no longer needed.